lib/StripeResource.js

```diff
@@ -134,13 +134,26 @@ StripeResource.prototype = {
       res.on('end', function() {
         var headers = res.headers || {};
 
+        // For convenience, make Request-Id easily accessible on
+        // lastResponse.
+        res.requestId = headers['request-id'];
+
+        var responseEvent = utils.removeEmpty({
+          api_version: req._requestEvent.api_version,
```

Comment:
Responses pass back an API version with them. Want to read this out of `Stripe-Version` just because it's likely to be the most accurate possible information?